src/test/java/address/util/PlatformSpecific.java

```diff
@@ -0,0 +1,100 @@
+package address.util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+
+public final class PlatformSpecific {
```

Comment:
I actually already created a class similar to this, named `OsDetector` in `util` of production since we have platform specific codes.